docs/hub/_toctree.yml

```diff
@@ -159,6 +159,8 @@
         title: ZenML on Spaces
   - local: spaces-embed
     title: Embed your Space
+  - local: spaces-run-with-docker
```

Comment:
Maybe also add to https://huggingface.co/docs/hub/index and https://huggingface.co/docs/hub/spaces